packages/twitter-verifier-circuits/scripts/3_gen_both_zkeys.sh

```diff
@@ -25,7 +25,7 @@ yarn
 echo "****GENERATING ZKEY 0****"
 start=$(date +%s)
 set -x
-NODE_OPTIONS='--max-old-space-size=56000' node ../../../node_modules/.bin/snarkjs groth16 setup "$R1CS_FILE" "$PHASE1" "$PARTIAL_ZKEYS"/"$CIRCUIT_NAME"_0.zkey -e=$ENTROPY1
+NODE_OPTIONS='--max-old-space-size=56000' node ../node_modules/.bin/snarkjs groth16 setup "$R1CS_FILE" "$PHASE1" "$PARTIAL_ZKEYS"/"$CIRCUIT_NAME"_0.zkey -e=$ENTROPY1
```

Comment:
can we define yarn snarkjs as `NODE_OPTIONS='--max-old-space-size=56000' node ../node_modules/.bin/snarkjs`? would make all this code a bit cleaner lol

Reply:
I tried this, but then the circuit.env need to be in the parent dir instead of `scripts` - so not sure 🤔